department-of-veterans-affairs/caseflow
# Justin/appeals 17515

Resolves [APPEALS-17515](https://vajira.max.gov/browse/APPEALS-17515)

### Description
Added an MST checkbox in a "Special issues" section on the non-rating add issue page for intaking an appeal

### Acceptance Criteria
- [ ] AC 1. Create the Frontend logic needed for the "None of these match, see more options" button selection to take the user to the PACT act and MST issues modal for the non-matching issue.
- [ ] AC 2. If the user selects the checkbox for MST and/or PACT, that selection status will be sent to the intake page when the user clicks the "add issue" button.
- [ ] AC 3. SPEC test written
- [ ] AC 4. Spec test validated

### Testing Plan
1. Go to the test users page and log in as an intake user (such as BVADWISE)
2. Click on mail intake and then choose start to begin an intake
3. Select "Decision Review Request ... 10182" and continue to search 
4. Choose a veteran (for example, 000100001)
5. Fill out the options on the page and continue to next step 
6. Click on "Add issue". If there are previous decision issues, the screen will ask if any match. Choose "None of these match, see more options"
7. Under "Issue description", there should be a label that says "Select any special issues that apply"
8. Following the label, a checkbox should be present labeled "Military Sexual Trauma (MST)"


### User Facing Changes

 BEFORE|AFTER
![image](https://user-images.githubusercontent.com/95875751/233464041-b09e7a2a-815d-4ed6-a872-269c04a8d1ed.png)
 ---|---
![image](https://user-images.githubusercontent.com/95875751/233464082-5538e58d-6ae1-427b-9895-f2d7d9b6e1d9.png)

### Code Documentation Updates
